src/main/java/gov/nasa/pds/tools/validate/BundleManager.java

```diff
@@ -504,10 +532,24 @@ public static void makeException(URL url, String location) {
         //     2.  Create a list of other bundle files to ignore so only the provided bundle is processed.
         //     3.  Create a list of collection files to ignore so only the latest collection file is processed.
 
+        LOG.debug("makeException:url,location {},{}",url,location);
+
         // First, find any other bundle files so they can be eliminated from crawling later.
         ArrayList<Target> otherBundleFiles = BundleManager.findOtherBundleFiles(url);
         BundleManager.m_ignoreList.addAll(otherBundleFiles);
 
+        // Because this function has knowledge of why a particular file is being ignored while crawling
+        // we will log it here.
+        if (otherBundleFiles != null) {
+            // Only print the size if we know that otherBundleFiles is not null.
+            LOG.debug("makeException:url,otherBundleFiles.size() {},{}",url,otherBundleFiles.size());
+            for (Target target : otherBundleFiles) {
+                LOG.info("SKIP: {} due to not being selected as the bundle target",target.getUrl());
```

Comment:
@qchaupds i see now that this is not appearing because this is being logged as an info message. I was hoping we would leverage the [SKIP status](https://github.com/NASA-PDS/validate/blob/main/src/main/java/gov/nasa/pds/validate/status/Status.java#L42) and log that in our logs as appropriate. looks like we use that here: https://github.com/NASA-PDS/validate/blob/main/src/main/java/gov/nasa/pds/validate/ValidateLauncher.java#L1344 . So something like that?

Reply:
@jordanpadams Please grab the lastest commit that has changes to 3 files to implement what you'd suggested.

It is odd that you weren't able to see any messages.  I am able to see the "SKIP:" both from the INFO logging and the System.println statements.  Perhaps my environment is different than yours.  Regardless, you should be able to see the  "SKIP:" messages now.

{pds-dev3.jpl.nasa.gov}/home/qchau/sandbox/validate 130 % validate -R pds4.bundle -v 0 --skip-context-reference-check --skip-content-validation -r report_github372_bundle_invalid.json -s json -t src/test/resources/github69/invalid/bundle_kaguya_derived.xml > & t10

{pds-dev3.jpl.nasa.gov}/home/qchau/sandbox/validate 131 % grep SKIP: t10                                                                                                                                  [main] INFO gov.nasa.pds.tools.validate.BundleManager - SKIP: file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/bundle_kaguya_derived_2.xml due to not being selected as the bundle target
SKIP: file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/bundle_kaguya_derived_2.xml due to not being selected as the bundle target
[main] INFO gov.nasa.pds.tools.validate.BundleManager - SKIP: file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory_2.xml due to logical_identifier the same as selected collection but different version
SKIP: file:/home/qchau/sandbox/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory_2.xml due to logical_identifier the same as selected collection but different version

Reply:
@qchaupds sorry this is not clear. Instead of adding these custom "SKIP" print statements or any of the other methods that are being custom built here, I would like us to use the the existing SKIP Status enumeration to actually tag these files as "SKIPPED" in the output reporting. https://github.com/NASA-PDS/validate/blob/main/src/main/java/gov/nasa/pds/validate/status/Status.java#L42 

By doing so, these will be reported out as part of our nominal reporting mechanisms (e.g. no change in verbosity needed, no debug flags needed, etc.). Here is a notional example of what I am thinking will be output when running validate with our nominal reporting. Note the output, and the count changes in the reporting:

**With no verbosity set (e.g. default output)**
```
$ validate-2.1.0-SNAPSHOT/bin/validate --skip-content-validation -R pds4.bundle -t src/test/resources/github69/invalid/bundle_kaguya_derived.xml
```
```
...
  PASS: file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory.xml
        4 product validation(s) completed

  SKIP: file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/github69/invalid/bundle_kaguya_derived_2.xml
        5 product validation(s) completed

  SKIP: file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory_2.xml
        6 product validation(s) completed
...
  Product Validation Summary:
    8          product(s) passed
    0          product(s) failed
    2          product(s) skipped

  Referential Integrity Check Summary:
    8          check(s) passed
    0          check(s) failed
    2          check(s) skipped
```

**With verbosity set to INFO to see why the files were skipped**
```
$ validate-2.1.0-SNAPSHOT/bin/validate --skip-content-validation -R pds4.bundle -t src/test/resources/github69/invalid/bundle_kaguya_derived.xml
```
```
...
  PASS: file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory.xml
        ...
        4 product validation(s) completed

  SKIP: file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/github69/invalid/bundle_kaguya_derived_2.xml
        INFO  [info.integrity.unreferenced_file]  Skipping file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/github69/invalid/bundle_kaguya_derived_2.xml
        due to not being selected as the bundle target
        5 product validation(s) completed

  SKIP: file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory_2.xml
        INFO  [info.integrity.unreferenced_file]  Skipping file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/github69/invalid/data_spectra/spectra_data_collection_inventory_2.xml
        due to logical_identifier the same as selected collection but different version
        6 product validation(s) completed
...
  Product Validation Summary:
    8          product(s) passed
    0          product(s) failed
    2          product(s) skipped

  Referential Integrity Check Summary:
    8          check(s) passed
    0          check(s) failed
    2          check(s) skipped
```

Reply:
@jordanpadams I am not able to see the difference between the "validate" commands for the two examples.
Are you missing the "-v 1" or something like that for the 2nd example?

Reply:
Yep. Sorry. Just updated below.